concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -2890,6 +2890,35 @@ jobs:
                   paas-billing-collector \
                   -f manifest-collector.yml
 
+      - task: paas-billing-acceptance-tests
+        config:
+          platform: linux
+          image_resource: *cf-acceptance-tests-image-resource
+          params:
+            AWS_REGION: ((aws_region))
+            SYSTEM_DNS_ZONE_NAME: ((system_dns_zone_name))
+            DEPLOY_ENV: ((deploy_env))
+          inputs:
+            - name: paas-cf
+            - name: paas-billing
+              path: src/github.com/alphagov/paas-billing
```

Comment:
We tried switching to `go mod` so that we didn't have to put the source inside a `GOPATH`, but it snowballed: about:blank